src/main/java/net/hollowcube/polar/PolarWorld.java

```diff
@@ -113,7 +115,11 @@ public void updateChunkAt(int x, int z, @NotNull PolarChunk chunk) {
 
     public enum CompressionType {
         NONE,
-        ZSTD;
+        ZSTD,
+        LZ4_FAST,
```

Comment:
Half of these are duplicates (in how they actually write). they should just not exist.